cmd/do/env.go

```diff
@@ -57,8 +57,8 @@ func env(cfg Config) *shell.Env {
 
 	path = append(path,
 		cfg.bin().System(),
-		cfg.JavaHome.Join("bin").System(),
 	)
+	path = append([]string{cfg.JavaHome.Join("bin").System()}, path...)
```

Comment:
I have multiple java version on build system and java 1.7 was used basically.
And I set java 1.8 installation path during ./do config but during the build, java 1.7 was used bacause of /usr/bin/java which is version 1.7
If you prefer keep original code, I can revert this change.
Thanks.